index.html

```diff
@@ -11,6 +11,7 @@
 <body>
   <div class="book-container row">
     <div class="book-image">
+      <img src="./assets/badge.png" alt="Book badge" class="book-badge">
```

Comment:
This is a decorative image, it doesn't bring any context to our layout or site. Therefore, we use `::after` or `::before`  pseudo classes to add these type of things instead of using a img tag. 

https://www.w3schools.com/css/css_pseudo_classes.asp

Reply:
Done